src/App.jsx

```diff
@@ -1,35 +1,45 @@
-import { useState } from 'react';
-import { Routes, Route } from 'react-router-dom';
-import Navbar from './components/Navbar/Navbar';
-import Dashboard from './components/Dashboard/Dashboard';
-import Landing from './components/Landing/Landing';
-import SignupForm from './components/SignupForm/SignupForm';
-import SigninForm from './components/SigninForm/SigninForm';
-import * as authService from '../src/services/authService';
+import { useState } from "react";
+import { Routes, Route, useNavigate } from "react-router-dom";
+import Navbar from "./components/Navbar/Navbar";
+import Dashboard from "./components/Dashboard/Dashboard";
+import Landing from "./components/Landing/Landing";
+import SignupForm from "./components/SignupForm/SignupForm";
+import SigninForm from "./components/SigninForm/SigninForm";
+import * as authService from "../src/services/authService";
+import CampaignForm from "./components/CampaignForm/CampaignForm";
 
 const App = () => {
   const [user, setUser] = useState(authService.getUser());
```

Comment:
need to initialize with an object

const [formData, setFormData] = useState({
    title: '',
    campaignType: '',
    description: '',
    goalAmount: '',
    endDate: ''
});